sirius/coding/encoders.py

```diff
@@ -96,9 +96,7 @@ def make(extra):
                 'name': 'add_device_encryption_key',
                 'params': {
                     'device_address': command.device_address,
-                    'encryption_key': base64.b64encode(
-                        claiming.key_from_claim_code(command.claim_code)
-                    ).decode('utf-8'),
+                    'encryption_key': claiming.key_from_claim_code(command.claim_code).decode('utf-8'),
```

Comment:
Hi @notjosh ! So this looks like the main change - not base64 encoding the encryption key... or was it double-base64'd before?

Reply:
This is the main change, yep! But, kind of neither of those :)

`claiming.key_from_claim_code` returns a base64'd string (via `process_claim_code`), so the key _does_ get base64'd, and that _was_ the only place for the base64-ing in the old Python 2 code. The double-base64-ing was introduced with the Python 3 migration via some bad copy/paste trying to avoid `bytes` vs `str` encodings.

I'd say the most correct solution here is to only do base64-ing when outputting the value in JSON, not in `process_claim_code`, but I didn't want to risk other side effects on a rushed fix. One of these days...